src/flows/pipes/Columns/readOnly.tsx

```diff
@@ -10,14 +10,17 @@ import {Hash, Mapping} from 'src/flows/pipes/Columns'
 // Contexts
 import {PipeContext} from 'src/flows/context/pipe'
 
+// Constants
+import {UNPROCESSED_PANEL_TEXT} from 'src/flows/constants'
+
 // Styles
 import './styles.scss'
 
 const Columns: FC<PipeProp> = ({Context}) => {
   const {data, loading, results} = useContext(PipeContext)
 
   if (!results.parsed || !results.parsed.table) {
-    let msg = 'This cell will display columns from the previous cell'
+    let msg = UNPROCESSED_PANEL_TEXT
```

Comment:
not sure if this change is relevant since it's a readonly component 🤷🏽

Reply:
i think it still needs to be here for how this one renders